persistence/src/main/java/cz/muni/fi/pa165/dao/HouseDaoImpl.java

```diff
@@ -41,30 +40,17 @@ public House getById(long id) {
 
     @Override
     public House getByName(String name) {
-        if(name == null)
-            throw new IllegalArgumentException("name cannot be null");
-
-        try {
-            return (House) em.createQuery("select house from House house where house.name = :name")
-                    .setParameter("name", name)
-                    .getSingleResult();
-        } catch (NoResultException ex) {
-            return null;
-        }
+        return (House) em.createQuery("select House from House house where House.name = :name")
```

Comment:
revertni zmeny na HouseDaoImpl a pullni si, nemen mi to pls - https://github.com/OndrejPontes/HauntedHouses/commit/3e0f36117383e73b0042834819d3974e6cef5d1c takto to chceme